scripts/build.ps1

```diff
@@ -270,6 +271,23 @@ function Publish-Package
         Write-Verbose "Move-Item $coreCLR20PackageDir\$file $coreCLRExtensionsDir -Force"
         Move-Item $coreCLR20PackageDir\$file $coreCLRExtensionsDir -Force
     }
+
+    # Publish Datacollector
+    $blameDataCollector = Join-Path $env:TP_ROOT_DIR "src\Microsoft.TestPlatform.BlameDataCollector\bin\$TPB_Configuration"
+    $blameDataCollectorNet46 = Join-Path $blameDataCollector $TPB_TargetFramework
+    $blameDataCollectorNetCore = Join-Path $blameDataCollector $TPB_TargetFrameworkCore
+    Copy-Item $blameDataCollectorNet46\* $fullCLRPackageDir -Force
+    Copy-Item $blameDataCollectorNetCore\* $coreCLR20PackageDir -Force
+
+    # Note Note: Move the Blame data collector to extensions folder
```

Comment:
Nit: Note appearing twice.
Why are copying and then moving them, why not directly copy to the required location?